homework-06/main.js

```diff
@@ -0,0 +1,115 @@
+// 1
+function compareNumbers(num1, num2, num3, num4) {
+    if ((Number.isNaN(num1)) || (Number.isNaN(num2)) || (Number.isNaN(num3)) || (Number.isNaN(num4))) {

```

Comment:
```suggestion
    if (isNaN(num1) || isNaN(num2) || isNaN(num3) || isNaN(num4)) {

```
Sometimes it's better to use `isNaN` as a separate method if we are not sure that the param is 100% number

<img width="566" alt="image" src="https://user-images.githubusercontent.com/84010501/191209972-7de71405-ca63-4653-be3b-94f3a8affcdc.png">